MvvmCross/Core/Core/Platform/LogProviders/NullLogProvider.cs

```diff
@@ -0,0 +1,7 @@
+namespace MvvmCross.Core.Platform.LogProviders
+{
+    internal sealed class NullLogProvider : MvxBaseLogProvider
```

Comment:
This should be removed, and instead set the logger to None.

Reply:
Without this class, the None option for logging will break any code which requests ILogProvider as an injected value. Do you have another alternative to this?